### PR TITLE
Generic component

### PR DESCRIPTION
**Pull request by benjaminxu251**

### CODE REVIEW

**Review by eneadodi — COMMENTED** [review]
Overall it looks good. However I would like you to switch the current editor with the Suneditor we have decided to use before approval

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/InformationItem.js
Comment (powellsl99) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/InformationItem.js:29]: You should look into where @Enoch-Hsiao implemented this in his PR, you can just copy paste a few things. Specifically he got it more intractable on mobile. Maybe that should be be put in its own file with the settings and then imported across the app so you dont have to keep copy pasting!

**Review by powellsl99 — CHANGES_REQUESTED** [review]
Looks great! Just one comment about sun editor

**Comment by Enoch-Hsiao** [comment]
Reopen generic component pr

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/InformationItemList.js
Comment (Enoch-Hsiao) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/InformationItemList.js]: Not yet, but this is where we will put the logic to save data to backend once we set it up.

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/InformationItem.js
Comment (Enoch-Hsiao) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/InformationItem.js]: Remove comments?

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/InformationItemList.js
Comment (Enoch-Hsiao) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/InformationItemList.js]: Change to function component if you can? If not we can do it in a separate pr.

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/pages/homepage.js
Comment (Enoch-Hsiao) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/pages/homepage.js:163]: Why?

Comment (benjaminxu251) [reply Ghost_in_the_Shell/client_folder_react_app/ethisim/src/pages/homepage.js:163]: I commented it out back when we weren't sure about using it; I'll delete it on my branch for now

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/pages/homepage.js
Comment (Enoch-Hsiao) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/pages/homepage.js]: Sorry for being strict but keep the copyright on the homepage XD

**Review by Enoch-Hsiao — DISMISSED** [review]
Looks good, add back copyright and resolve the conflict.

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/pages/homepage.js
Comment (benjaminxu251) [reply Ghost_in_the_Shell/client_folder_react_app/ethisim/src/pages/homepage.js]: lmao aight

**Review by Enoch-Hsiao — DISMISSED** [review]

**Review by powellsl99 — APPROVED** [review]
LGTM!

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/Logistics.js
Comment (powellsl99) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/Logistics.js]: Actually before you merge, can you fix this

**Review by powellsl99 — CHANGES_REQUESTED** [review]

**Review by powellsl99 — APPROVED** [review]
Looks good thanks for implementing the changes! Nice job! 
As Enoch said: mergeeee

**Pull request merged**
